### PR TITLE
refactor,chore(egen) : added delete experiment code in cleanup section and required packages in installation section 

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes

changed variable REGION to LOCATION
added matplotlib and pandas in installation section
added delete experiment code in cleanup section
performed linter test
followed template guidelines
**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb [hunk 1]
```diff
@@ -95,7 +98,7 @@
       "source": [
         "### Dataset\n",
         "\n",
-        "In this notebook, we will train a simple distributed neural network (DNN) model to predict automobile's miles per gallon (MPG) based on automobile information in the [auto-mpg dataset](https://www.kaggle.com/devanshbesain/exploration-and-analysis-auto-mpg)."
+        "In this notebook, we  train a simple distributed neural network (DNN) model to predict automobile's miles per gallon (MPG) based on automobile information in the [auto-mpg dataset](https://www.kaggle.com/devanshbesain/exploration-and-analysis-auto-mpg)."
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:101]: change "we" to "you":

- "In this notebook, you  train a simple distributed neural network ..."

**Review by kittyabs — COMMENTED** [review]
small edit

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb [hunk 2]
```diff
@@ -140,140 +140,87 @@
...
-        "id": "oM1iC_MfAts1"
-      },
-      "outputs": [],
-      "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
-        "\n",
-        "# Set the project id\n",
-        "! gcloud config set project $PROJECT_ID"
+        "    app = IPython.Application.instance()\n",
+        "    app.kernel.do_shutdown(True)"
```
Comment (katiemn) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:174]: Please add the restart warning from the template after this code cell

File: notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb [hunk 3]
```diff
@@ -140,140 +140,98 @@
...
       "metadata": {
-        "id": "457c78b08293"
+        "id": "bf2dc7995ac2"
       },
-      "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "<div class=\"alert alert-block alert-warning\">,\n",
+        "<b>⚠️ The kernel is going to restart. Wait until it's finished before continuing to the next step. ⚠️</b>,\n",
+        "</div>"
```
Comment (katiemn) [inline notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:212]: Please move this warning message to just under the Restart runtime code section (see notebook template for example)

**Review by katiemn — APPROVED** [review]

**Pull request merged**